packages/lesswrong/components/form-components/ImageUploadDefaultsDialog.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+
+const styles = (theme: ThemeType): JssStyles => ({
+  images: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'space-evenly'
+  },
+  image: {
+    padding: '5px 5px 2px',
+    margin: 3,
+    border: '2px solid transparent',
+    cursor: 'pointer',
+    '&:hover': {
+      borderColor: theme.palette.primary.main,
+    },
+  },
+})
+
+const ImageUploadDefaultsDialog = ({ onSelect, onClose, classes }: {
+  onSelect: Function,
+  onClose: ()=>void,
+  classes: ClassesType,
+}) => {
+
+  const { LWDialog, CloudinaryImage } = Components
+  
+  const selectImg = (img) => {
+    onSelect(img)
+    onClose()
+  }
+  
+  // these are EA Forum-specific
+  const defaultImages = [
```

Comment:
Maybe I should move this list elsewhere? Though this file is not that long so maybe it's fine

Reply:
It is in a file dedicated to default images, so I think I lean towards it being ok.